### ISSUE TITLE
App volume control

### ISSUE THREAD

**Issue by bige11** [xou816/spot#77]
It would be very convenient to have a slider integrated in the app to control the app volume independently from the general volume (I often lower it to prevent missing out other desktop/apps sounds, but I have to go through the audio settings - using Linux Mint 20).

P.S. Love the project! Merci

**Comment by xou816** [comment]
Hi! Thank you for the kind words :)
It is not a top priority of mine at the moment (I tried implementing it in the past and got stuck) but I'll leave this issue open for later :)

**Comment by Julioevm** [comment]
I'd like this as well. Is there a way to control the volume of an application externally?

**Comment by TomaszGasior** [comment]
In GNOME Settings, in Sound panel. You must be playing some music while changing volume, otherwise "librespot" item is not visible.

**Comment by Julioevm** [comment]
Also found this handy extension to do it from the task bar: https://extensions.gnome.org/extension/3499/application-volume-mixer/

**Comment by Diegovsky** [comment]
I implemented this via MPRIS in my branch today. @xou816 would you like me to open a PR?

**Comment by xou816** [comment]
Of course, if you feel like sharing it then a PR is welcome :)

**Comment by Diegovsky** [comment]
Great! I'll clean some stuff and open it up soon

**Comment by OKPersson** [comment]
Any updates on this? was the only real "issue" that I found when I installed the app, great app otherwise.

**Comment by Diegovsky** [comment]
Yes, let me give you a tldr: Spot supports song volume, but we don't have a slide for that yet. If you use MPRIS, you can change the volume from there.

Here's how I do it from gnome, using the [mpris indicator](https://extensions.gnome.org/extension/1379/mpris-indicator-button/) extension:
![image](https://user-images.githubusercontent.com/46163903/173150118-3693e1fd-c0fc-446c-9a53-f92a6161d7e5.png)

**Comment by OKPersson** [comment]
Perfect!, thanks for the update.

**Comment by Diegovsky** [comment]
I have been fiddling with adding a slider this morning but I have no idea where I should put it.

![image](https://user-images.githubusercontent.com/46163903/173199550-4e0ac7ab-59ae-46e3-8ec5-1467ad543aea.png)

Some notes:
- The volume icon is just a placeholder. 
- I looked into what Amberol does but it has a very different layout compared to Spot.
- I'm trying to embed the slider in the controls, like Spotify does, to see if it looks better.

**Comment by OKPersson** [comment]
Cool, I think that it would be best located roughly the same as the default Spotify app. However I don't know how that would work with smaller layouts such as the mobile view.
